scipy/spatial/distance.py

```diff
@@ -934,6 +934,47 @@ def kulsinski(u, v, w=None):
 
 
 def kulczynski1(u, v, w=None):
+    """
+    Compute the Kulczynski 1 dissimilarity between two boolean 1-D arrays.
+
+    The Kulczynski 1 dissimilarity between two boolean 1-D arrays `u` and `v`,
+    is defined as
+
+    .. math::
+
+         \\frac{c_{TT}}
+              {c_{FT} + c_{TF}}
+
+    where :math:`c_{ij}` is the number of occurrences of
+    :math:`\\mathtt{u[k]} = i` and :math:`\\mathtt{v[k]} = j` for
+    :math:`k < n`.
+
+    Parameters
+    ----------
+    u : (N,) array_like, bool
```

Comment:
Can you add an example with using `bool`? Is this vectorized?

Reply:
`_nbool_correspond_all` is vectorized. So, seems like yes, this function is vectorized as well. I have followed pattern similar to the other functions in this file.